solution/backend/regcore/search/models.py

```diff
@@ -21,28 +20,25 @@ def search(self, query):
         if query and query.startswith('"') and query.endswith('"'):
             search_type = "phrase"
             cover_density = True
+        search_query = SearchQuery(query, search_type=search_type, config='english')
 
-        return self\
+        return self.annotate(vector_column=RawSQL("vector_column", [], output_field=SearchVectorField()))\
```

Comment:
Impossible.  Django doesnt allow you to do that otherwise.  Vector column is a calculated field on the search_searchindexv2 table.  Normally when we annotate a column its part of a model but django does not and will not support calculated fields formed by postgres.  We arent doing any sort of calculations here, just pulling a value that exist on the table that django does not know about which is associated to the model

Reply:
Its a read-only field and there are no inputs or calculations pull it in so it would be fine by from any sort of security standpoint.  There is no input a person can do to change it.